shared/src/store/action-reducers/simulated-region.ts

```diff
@@ -61,8 +61,8 @@ export class ResizeSimulatedRegionAction implements Action {
 }
 
 export class RenameSimulatedRegionAction implements Action {
-    @IsValue('[SimulatedRegion] Rename simulatedRegion' as const)
```

Comment:
This needs a migration

Reply:
Done in 721f72b0fa9ffccfbdcfc2af642b3f90476ce1c2
Did I miss anything?